Makefile

```diff
@@ -108,8 +108,9 @@ else
 	$(call MINIKUBE_GO_BUILD_CMD,$@,$*)
 endif
 
-out/e2e-%-amd64:
-	GOOS=$* GOARCH=amd64 go test -c k8s.io/minikube/test/integration --tags=integration -o $@
+.PHONY: e2e-%-amd64
+e2e-%-amd64:
```

Comment:
Ah I see, would it make sense then to have a rule to actually run the test, instead of to compile it to a binary? Then it'll recompile each time you run it but it should be pretty quick.